test/connector/tcp/mssql/remote_debug.md

```diff
@@ -0,0 +1,47 @@
+# Using the Secretless-Broker Remote Debug Image
```

Comment:
can we make this just README.md in this directory? for now, it can just include remote debug instructions - but having this here would make it easier for devs to add other mssql-specific dev notes, and that can be quite handy later

Reply:
I think with that these changes could go in

Reply:
Okay, I moved the remote-debug.md to README.md, and added an intro into the README.md to describe what remote debugging is.